### PR TITLE
Add wstETH BASE

### PR DESCRIPTION
**Pull request by MirthFutures**
**Description**

A clear and concise description of the features you're adding in this pull request.

**Tests**

Please describe any tests you've added. If you've added no tests, or left important behavior untested, please explain why not.

**Additional context**

Add any other context about the problem you're solving.

**Metadata**

- Fixes #[Link to Issue]

### CODE REVIEW

**Comment by MirthFutures** [comment]
Added, sorry about that miss

File: data/wstETH/data.json [hunk 1]
```diff
@@ -16,6 +17,12 @@
...
         "bridge": "0x8E01013243a96601a86eb3153F0d9Fa4fbFb6957"
       }
-    }
+    },
+    "base": {
+      "address": "0xc1CBa3fCea344f92D9239c08C0568f6F2F0ee452",
+      "overrides": {
+        "bridge": "0xac9D11cD4D7eF6e54F14643a393F68Ca014287AB"
+      }
+    },
```
Comment (roberto-bayardo) [inline data/wstETH/data.json:26]: drop the comma
```suggestion
    }
```

**Comment by TheDZhon** [comment]
For the reference:
- https://snapshot.org/#/lido-snapshot.eth/proposal/0x8b35f64fffe67f67d4aeb2de2f3351404c54cd75a08277c035fa77065b6792f4
- https://docs.lido.fi/deployed-contracts/#base

**Comment by GabrielaMelendezQuan** [comment]
Hi @MirthFutures,
Can you please provide the following information for each of these two addresses? [0xac9D11cD4D7eF6e54F14643a393F68Ca014287AB](https://basescan.org/address/0xac9D11cD4D7eF6e54F14643a393F68Ca014287AB) and [0x0E37599436974a25dDeEdF795C848d30Af46eaCF](https://basescan.org/address/0x0E37599436974a25dDeEdF795C848d30Af46eaCF)

1. The nature of any multi-sig or other custody setup (# signers required / total signers, or other guarded access)
2. A claim indicating how many are independent signers (i.e. different individuals) - ideally please indicate the addresses of the multisig participants and the associated holder identities for those addresses
3. Whether you use a hardware wallet, and if not where the keys are stored
4. Any 3rd party audit from a reputable auditor that the requirements have been met, and/or provide access to Coinbase so that Coinbase can validate the controls (by providing us with the multi-sig address to verify on chain)
5. A commitment to providing notification of any changes to this validated, audited process

**Comment by kadmil** [comment]
gm gm @GabrielaMelendezQuan!

The bridging contract's [0xac9D11cD4D7eF6e54F14643a393F68Ca014287AB](https://basescan.org/address/0xac9D11cD4D7eF6e54F14643a393F68Ca014287AB) upgradability is managed by a dedicated smart contract called OptimismBridgeExecutor at [0x0E37599436974a25dDeEdF795C848d30Af46eaCF](https://basescan.org/address/0x0E37599436974a25dDeEdF795C848d30Af46eaCF).

This setup allows the upgrade authority to lay squarely on Lido DAO Aragon Agent contract [0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c](https://etherscan.io/address/0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c) (check out the address at [Lido docs](https://docs.lido.fi/deployed-contracts/#lido-v2-dao-contracts)), setting the upgradability through the cross-chain governance motion passing. That means no multisigs are connected to the regular operation of the bridging parts on Base or on mainnet.

That said, one more thing to note is that there's a dedicated Emergency Brakes multisig with the roles to pause deposits or withdrawals on either mainnet or Base side of the bridging parts (`DEPOSITS_DISABLER_ROLE` & `WITHDRAWALS_DISABLER_ROLE`). Said multisig has public and Lido DAO-vetted setup (latest update vote with all relevant details can be found here: https://snapshot.org/#/lido-snapshot.eth/proposal/0xfe2a6a6506a642b616118363bc29aa83dd9ef2ec80447bb607a8f52c0a96aed0; follow the links to discussion for deeper context).

As for the audits, the code used has been audited under the project of enabling WstETH bridging to Optimism (the audit can be found here: https://github.com/lidofinance/audits#08-2022-oxorio-governance-crosschain-bridges-smart-contracts-security-audit-report) & verified that the Base deployment uses the same code: https://research.lido.fi/t/wsteth-deployment-to-base-and-ownership-acceptance-by-lido-dao/5668/5

**Review by roberto-bayardo — DISMISSED** [review]
looks good from Base side.

**Review by hamdiallam — APPROVED** [review]

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**